packages/mobile/src/web3/saga.ts

```diff
@@ -230,8 +235,34 @@ export async function readPrivateKeyFromLocalDisk(
 ): Promise<string> {
   const filePath = getPrivateKeyFilePath(account)
   Logger.debug('readPrivateKeyFromLocalDisk', `Reading private key from ${filePath}`)
-  // TODO(ashishb): Read and decrypt private key instead
-  return RNFS.readFile(getPrivateKeyFilePath(account))
+  const hexEncodedEncryptedData: string = await RNFS.readFile(filePath)
+  const encryptedDataBuffer: Buffer = new Buffer(hexEncodedEncryptedData, 'hex')
```

Comment:
@jmrossy On the other hand, I find this explicit typing to help with readability in cases that switch between strings/buffers/etc
Some linting rules will prevent explicit typing for common types like numbers, strings (ex: https://palantir.github.io/tslint/rules/no-inferrable-types/) but in the case of less common types I find the explicit typing helpful